packages/grid/_modules_/grid/constants/eventsConstants.ts

```diff
@@ -20,13 +20,20 @@ export const GRID_CELL_OVER = 'cellOver';
 export const GRID_CELL_OUT = 'cellOut';
 export const GRID_CELL_ENTER = 'cellEnter';
 export const GRID_CELL_LEAVE = 'cellLeave';
+export const GRID_CELL_KEYDOWN = 'cellKeyDown';
+
+export const GRID_CELL_ENTER_EDIT = 'cellEnterEdit';
+export const GRID_CELL_EXIT_EDIT = 'cellExitEdit';
+export const GRID_CELL_NAVIGATION_KEYDOWN = 'cellNavigationKeyDown';
 
 export const GRID_ROW_CLICK = 'rowClick';
 export const GRID_ROW_DOUBLE_CLICK = 'rowDoubleClick';
 export const GRID_ROW_OVER = 'rowOver';
 export const GRID_ROW_OUT = 'rowOut';
 export const GRID_ROW_ENTER = 'rowEnter';
 export const GRID_ROW_LEAVE = 'rowLeave';
+export const GRID_ROW_EDIT_MODEL_CHANGE = 'editRowModelChange';
+export const GRID_ROW_SELECTED = 'rowSelected';
```

Comment:
I think that events should be active, not passive. It might be better with:
```suggestion
export const GRID_ROW_SELECT = 'rowSelect';
```

Reply:
that a BC, separate concern I think.